openmetadata-spec/src/main/resources/json/schema/entity/services/connections/pipeline/airbyteConnection.json

```diff
@@ -26,6 +26,17 @@
       "type": "string",
       "format": "uri"
     },
+    "username": {
```

Comment:
let's add this to the 0.13 doc about the updates on the connections

Reply:
sure will do that with #8502